GDV-HERBST/appendix/Klausur.md

```diff
@@ -0,0 +1,43 @@
+# Themenübersicht Klausur
+
+## CAD / CAE / Hardware
+
+- Begriffe aus dem CAx-Umfeld
+- Unterstützung durch CAx-Techniken im Entwicklungsprozess
+- Produktdatenmodell, PDM-, PLM-Systeme
+- Produktentwicklung, Produktdatenmanagement, Produktlebenszyklus
+- virtuelles Produkt, digitaler Zwilling
+- Aufbau von CAD-Systemen
+- Besonderheiten CAD-Rechner, E/A-Komponenten, Betriebssysteme für CAD-Arbeitsplätze
+- CAD-Kernel und –Systeme, Applikationen
+- Grafikstandards- bzw. -schnittstellen
+- Volumenmodelle
+- Boolesche Operatoren, Formelemente, CSG
+- Tesselierung, Level of Detail
+- Skizziertechniken und parametrische Konstruktion
+- Modellbaum, -historie
+- Bauteile und Baugruppen
+- Virtuelle Prototypen, Rapid Prototyping/3D-Druck, Additive Manufacturing
+- Grundlegende Vorgehensweise Modellierung mit Inventor, Import/Export
+
+## Grafische Datenverarbeitung
+
+- Objekt- und Bildraum
+- 2D-, 3D-Grafikpipeline
+- Bildauflösung, Bildschirmgrößen, Umrechnungen, Seitenverhältnisse
+- Probleme bei Rastergeräten zur Ausgabe:
```

Comment:
ich glaub hier müssen ein paar Punkte eingerückt werden

Reply:
please elaborate ❓

Reply:
In Zeile 28 steht am Ende ein Doppelpunkt.
Ich gehe demnach davon aus, dass ein paar der Zeilen darunter dem Punkt untergeordnet sein sollen.